ephios/core/forms/users.py

```diff
@@ -52,6 +52,7 @@
     "core.add_qualification",
     "core.change_qualification",
     "core.delete_qualification",
+    "auth.publish_event_for_group",
```

Comment:
I'm surprised to see the `auth.` here. It's because it's a permissions groups have on other groups, and the Group model is based in django's auth app?

Reply:
exactly